Unix/server/GNUmakefile

```diff
@@ -18,7 +18,7 @@ INCLUDES = $(TOP) $(TOP)/common
 
 DEFINES = HOOK_BUILD MI_CONST=
 
-LIBRARIES = wsman xmlserializer http protocol sock disp provreg provmgr $(MIAPILIB) wql base xml $(PALLIBS) $(INDIMGRLIB) omi_error $(INDICMNLIB) micodec mofparser
+LIBRARIES = protocol sock $(MIAPILIB) wql base $(PALLIBS) $(INDIMGRLIB) omi_error $(INDICMNLIB) micodec mofparser
```

Comment:
other things that should be able to go include:
* miapi, wql, micodec, mofbase
things I would also expect to be able to go include:
* indicationmgr and omi_error

Reply:
Removed